kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -284,9 +303,8 @@
           const viewPort = firstPage.getViewport({ scale: 1 });
           this.firstPageHeight = viewPort.height;
           this.firstPageWidth = viewPort.width;
+          this.scale = this.elementWidth / (this.firstPageWidth * this.screenSizeMultiplier);
```

Comment:
I have left the scale calculation here with `this.elementWidth` instead of `this.refs.pdfContainer` because at this time (on created() hook) the content has not yet been rendered.